Dockerfile

```diff
@@ -41,5 +41,6 @@ RUN cd /infrastructure/ansible && ansible-galaxy install -r requirements.yml
 ADD . /infrastructure
 WORKDIR /infrastructure
 
+LABEL com.circleci.preserve-entrypoint=true
```

Comment:
This looks like a Circle CI feature https://circleci.com/docs/2.0/custom-images/#adding-an-entrypoint
Ok.